rewriter/tests/ro_sharing/plugin.c

```diff
@@ -35,14 +37,15 @@ const uint32_t *get_plugin_uint(bool secret) {
 // LINKARGS: --wrap=read_main_string
 void read_main_string(const char *str) {
   // Check that we can read a string passed from main
-  LOG("%s", str);
+  cr_log_info("%s", str);
 }
 
 // LINKARGS: --wrap=read_main_uint
 void read_main_uint(const uint32_t *shared, const uint32_t *secret) {
   // Check that we can read a pointer to rodata passed from main
-  LOG("0x%x", *shared);
+  cr_log_info("0x%x", *shared);
 
   // Check that we can't read a pointer to data passed from main
+  // TODO can we change this LOG to cr_log_info?
   CHECK_VIOLATION(LOG("0x%x", *secret));
```

Comment:
Yes, change this to a cr_log and put the `CHECK_VIOLATION` around the `*secret` if we can?